ethereum/executionlayer/src/main/java/tech/pegasys/teku/ethereum/executionlayer/MilestoneBasedEngineJsonRpcMethodsResolver.java

```diff
@@ -120,6 +123,10 @@ private Map<EngineApiMethod, EngineJsonRpcMethod<?>> electraSupportedMethods() {
     return methods;
   }
 
+  private Map<EngineApiMethod, EngineJsonRpcMethod<?>> fuluSupportedMethods() {
+    return new HashMap<>();
```

Comment:
Not sure on this, it means no supported methods for FULU. Are you sure?

Reply:
No. Tt means we are not adding any FULU specific methods.

Reply:
Not sure. Adding test were suggested